src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java

```diff
@@ -142,6 +162,50 @@ private String generateQuery(String partitionFromDate, String partitionToDate, S
     return String.format(queryTemplate, tableName, condition.toString());
   }
 
+  private String generateQueryForMaterializingView(String dataset, String table, String filter) {
+    String queryTemplate = "select * from %s %s";
+    StringBuilder condition = new StringBuilder();
+
+    if (!Strings.isNullOrEmpty(filter)) {
+      condition.append(String.format(" where %s", filter));
+    }
+
+    String tableName = dataset + "." + table;
+    return String.format(queryTemplate, tableName, condition.toString());
+  }
+
+
+  /**
+   * Create {@link TableReference} to export Table or View
+   *
+   * @param type BigQuery table type
+   * @param inputProjectId project id of source table
+   * @param datasetId dataset id of source table
+   * @param tableId The ID of the table
+   * @param configuration Configuration that contains View Materialization ProjectId and View
+   *     Materialization Dataset Id
+   * @return {@link TableReference}
+   */
+  private TableReference createExportTableReference(
+      Type type,
```

Comment:
nit: please fix indentation here and in other places in this PR.  You can intellij settings file from https://wiki.cask.co/display/CE/Coding+Standard.

Reply:
@rmstar Created PR agains develop: https://github.com/data-integrations/google-cloud/pull/263